test/unit/effects.js

```diff
@@ -1178,45 +1178,45 @@ test("animate with per-property easing", function(){
 test("animate with CSS shorthand properties", function(){
 	expect(11);
 
-	var _default_count = 0,
-		_special_count = 0,
+	var default_count = 0,
+		special_count = 0,
 		propsBasic = { "padding": "10 20 30" },
-		propsSpecial = { "padding": [ "1 2 3", "_special" ] };
+		propsSpecial = { "padding": [ "1 2 3", "special" ] };
 
-	jQuery.easing._default = function(p) {
+	jQuery.easing.default = function(p) {
```

Comment:
Let's name this to better match its purpose, something like `jQuery.easing.animationScope` (and the other one to e.g. `jQuery.easing.propertyScope`).